src/app/metric/github/metric-config.tsx

```
@@ -0,0 +1,213 @@
+  const [metricName, setMetricName] = useState(template?.label ?? "");
+
+  if (!template) return <div>Template not found</div>;
+
+  // Find if there's a dynamic-select for repos
+  const repoParam = template.requiredParams.find(
+    (p) => p.type === "dynamic-select" && p.name === "REPO",
+  );
+
+  // Fetch repos for dynamic dropdown
+  const { data: reposData, isLoading: isLoadingRepos } =
+    api.metric.fetchIntegrationData.useQuery(
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)